WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageParentViewModel.kt

```diff
@@ -49,7 +50,7 @@ class PageParentViewModel
         }
     }
 
-    private fun loadPages(pageId: Long) = launch(CommonPool) {
+    private fun loadPages(pageId: Long) = GlobalScope.launch(CommonPool) {
```

Comment:
I see warnings that `CommonPool` is deprecated and it should be replaced with `Dispatchers.Default`.